java/src/test/java/org/rocksdb/PerfContextTest.java

```diff
@@ -80,6 +81,8 @@ public void testAllGetters() throws RocksDBException, IntrospectionException,
 
   @Test
   public void testGetBlockReadCpuTime() throws RocksDBException {
+    Assume.assumeFalse("We are not running this test on OpenBSD or Windows",
```

Comment:
Why is this test now prohibited on Windows? Presumably it was passing Windows CI before?
If this is memory limited, perhaps again we should change the assumption to `!Environment.is64Bit()`?

Reply:
I found it when I was developing this issues. We never run it on Windows. There are some test which we don't run via CMake.

Reply:
Okay thanks @rhubner, as agreed we will address the Windows CMake tests issue in a separate PR.